test/util/core/iterable-to-string-test.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+var referee = require("referee");
+var iterableToString = require("../../../lib/sinon/util/core/iterable-to-string");
+var assert = referee.assert;
+
+describe("util/core/iterable-to-string", function () {
+    it("returns an String representation of Array objects", function () {
+        var arr = [1, "one", true, undefined, null];
+        var expected = "1,'one',true,undefined,null";
+
+        assert.equals(iterableToString(arr), expected);
+    });
+
+    it("returns an String representation of Map objects", function () {
+        if (typeof Map === "function") {
+            var map = new Map();
+            map.set(1, 1);
+            map.set("one", "one");
+            map.set(true, true);
+            map.set(undefined, undefined);
+            map.set(null, null);
+            var expected = "[1,1]," +
+                "['one','one']," +
+                "[true,true]," +
+                "[undefined,undefined]," +
+                "[null,null]";
+
+            assert.equals(iterableToString(map), expected);
+        }
+    });
+
+    it("returns an String representation of Set objects", function () {
+        if (typeof Set === "function") {
```

Comment:
Same question as above regarding wrapping the `it`statement with the conditional